src/itest/kotlin/nl/info/zac/itest/ZaakKoppelenRestServiceTest.kt

```diff
@@ -44,12 +46,24 @@ class ZaakKoppelenRestServiceTest : BehaviorSpec({
                 zaakUUID = getString("uuid").let(UUID::fromString)
             }
         }
+        itestHttpClient.performGetRequest(
+            "$ZAC_API_URI/zaken/zaak/id/ZAAK-2000-0000000003"
+        ).use { getZaakResponse ->
+            val responseBody = getZaakResponse.body!!.string()
+            logger.info { "Response: $responseBody" }
+            with(JSONObject(responseBody)) {
+                teKoppelenZaakUuid = getString("uuid").let(UUID::fromString)
+            }
+        }
 
-        When("searching for a HOOFDZAAK linkable zaken with 'ZAAK-2000' zaak identifier") {
+        When(
+            "searching for a DEELZAAK linkable zaken with 'ZAAK-2000' zaak identifier and then link teh linkable " +
+                "zaak"
+        ) {
```

Comment:
```suggestion
        When(
            """
            searching for a DEELZAAK linkable zaken with 'ZAAK-2000' zaak identifier and then link the linkable
            zaak
            """.trimIndent()
        ) {
```

You IDE should warn you about spelling mistakes like `teh` ?